sky/adaptors/cloudflare.py

```diff
@@ -92,7 +123,7 @@ def client(service_name: str, region):
     # Reference: https://stackoverflow.com/a/59635814
 
     session_ = session()
-    cloudflare_credentials = session_.get_credentials().get_frozen_credentials()
+    cloudflare_credentials = get_r2_credentials()
```

Comment:
For `client()` and `resource()`, can we pass in `session_` as an argument to `get_r2_credentials()`?